### ISSUE TITLE
Write ZIM files in parallel if multiple --format arguments

### ISSUE THREAD

**Issue by kelson42** [openzim/mwoffliner#701]
Currently (except if `--useCache` is setup) if `--format=novid --format=nopic` is used then we have for the same article two HTTP requests to the API. This is because the two ZIM files are written in a row. If they would be written in parallel, then if would be possible to mutualise the HTTP request and save network resource. Of course all of this should also work properly if `--useCache` is set.

This has been first discussed in #695

**Comment by ISNIT0** [comment]
> Of course all of this should also work properly if --useCache is set.

Surely if we were writing in parallel there would be no need for `--useCache`?

**Comment by kelson42** [comment]
Seems really complicated to impemented with little added value. Will close as WONTFIX.
